src/rev_ai/models/language_id/language_id_job.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+"""Job model"""
+
+from ..asynchronous.job_status import JobStatus
+
+
+class LanguageIdentificationJob:
+    def __init__(
+            self, id_, created_on, status,
+            completed_on=None,
+            callback_url=None,
+            metadata=None,
+            media_url=None,
+            failure=None,
+            failure_detail=None,
+            processed_duration_seconds=None,
+            delete_after_seconds=None):
+        """
+        :param id_: unique id of job
+        :param created_on: date and time at which this job was started
+        :param status: current job status 'IN_PROGRESS', 'COMPLETED',
+                       or 'FAILED'
+        :param completed_on: date and time at which this job finished
+                             being processed
+        :param callback_url: callback_url if provided
+        :param metadata: metadata if provided
+        :param media_url: url of transcribed media if job was submitted
```

Comment:
dont we also provide fileName or something If submitted multi part?

Reply:
we do not for language id